src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx

```diff
@@ -1,64 +1,81 @@
 ---
-title: Troubleshoot AJAX data collection
+title: Troubleshoot Ajax data collection
```

Comment:
`AJAX` is an acronym, so it's better displayed in all caps when trying to be "proper"